cfg/common_test.go

```diff
@@ -53,7 +53,10 @@ monitoring:
         start: 0.06
         bucketsNum: 60
         bucketSize: 3
-
+	findDurationLinHistogram:
+        start: 0.5
+        bucketsNum: 20
+        bucketSize: 0.5
```

Comment:
It would be good if the numbers in the test were different from defaults. Then we can actually test the parsing.